README.md

```diff
@@ -36,21 +36,47 @@ $ make
 ```
```

Comment:
Line 24: pre-built binary should be found in [plugin releases](https://github.com/intelsdi-x/snap-plugin-processor-statistics/releases) or in [plugin catalog](https://github.com/intelsdi-x/snap/blob/master/docs/PLUGIN_CATALOG.md#processors)? @nanliu